src/mavlink-router/logendpoint.h

```diff
@@ -55,16 +55,19 @@ class LogEndpoint : public Endpoint {
     LogMode _mode;
 
     Timeout *_logging_start_timeout = nullptr;
+    Timeout *_logging_stop_timeout = nullptr;
     Timeout *_alive_check_timeout = nullptr;
     uint32_t _timeout_write_total = 0;
 
     virtual const char *_get_logfile_extension() = 0;
 
     void _send_msg(const mavlink_message_t *msg, int target_sysid);
     void _remove_start_timeout();
+    void _remove_stop_timeout();
     bool _start_alive_timeout();
 
     virtual bool _start_timeout() = 0;
+    virtual bool _stop_timeout() {return true;}
```

Comment:
I'd enforce this to be implemented.
```suggestion
    virtual bool _stop_timeout() = 0;
```

Reply:
@bkueng Ok, this means that I will have to go through the other logging implementations and see how they function.